src/client/index.js

```diff
@@ -202,8 +202,27 @@ const getContextParams = args => {
   }
 };
 
+const getNPlurals = (...args) => {
+  // Only true in tests
+  if (args.length !== 1) {
+    return null;
+  }
```

Comment:
why is that needed?

Reply:
Same reason as below. In the tests, we have e.g.
```js
const {Translate, PluralTranslate} = makeComponents(
    msg => msg,
    (msg, msgpl, n) => (n === 1 ? msg : msgpl)
);
```